packages/backend/src/controllers/schedulerController.ts

```diff
@@ -111,6 +111,36 @@ export class SchedulerController extends Controller {
         };
     }
 
+    /**
+     * Set scheduler active/inactive
+     * @param schedulerUuid The uuid of the scheduler to update
+     * @param req express request
+     * @param body the active flag
+     */
+    @Middlewares([
+        allowApiKeyAuthentication,
+        isAuthenticated,
+        unauthorisedInDemo,
+    ])
+    @SuccessResponse('201', 'Updated')
+    @Patch('{schedulerUuid}')
+    @OperationId('updateSchedulerActive')
+    async patchActive(
```

Comment:
alternative implementation: 

UPdate existing udpate endpoint to accept `active` flag

Const: , for disabling a scheduler, we will require to pass the full object